axelor-account/src/main/java/com/axelor/apps/account/web/InvoiceTermController.java

```diff
@@ -138,6 +137,15 @@ public void initInvoiceTerm(ActionRequest request, ActionResponse response) {
             }
           }
         }
+      } else if (request.getContext().get("_invoiceId") != null) {
+        invoice =
+            Beans.get(InvoiceRepository.class)
+                .find(Long.valueOf((Integer) request.getContext().get("_invoiceId")));
+      }
+
+      if (invoice != null) {
+        invoiceTermService.initCustomizedInvoiceTerm(invoice, invoiceTerm);
+        response.setValues(invoiceTerm);
```

Comment:
Please refactor the whole `initInvoiceTerm` method as this is becoming too complex.
Please change it so we avoid nested `if` conditions: first retrieve invoice/move, then depending on which variable is null, call the right service.
Also I don't get why we call `moveLine.setMove(move);` so please check if this is really necessary.

Reply:
initInvoiceTerm method refactored and remotion of `moveLine.setMove(move);`